src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -185,6 +204,30 @@ public SchemaShape GetOutputSchema(SchemaShape inputSchema)
     /// Produces a bag of counts of ngrams (sequences of consecutive words of length 1-n) in a given text.
     /// It does so by hashing each ngram and using the hash value as the index in the bag.
     /// </summary>
+    /// <summary>
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="ITransformer"/>.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | Vector of [Text](<xref:Microsoft.ML.Data.TextDataViewType>) |
+    /// | Output column data type | Vector of known-size of <xref:System.Single> |
+    ///
+    /// The resulting <xref:Microsoft.ML.ITransformer/> creates a new column, named as specified in the output column name parameters, and
+    /// produces a vector of counts of n-grams (sequences of consecutive words of length 1-n) from a given data.
+    /// It does so by hashing each ngram and using the hash value as the index in the bag.
+    ///
+    /// <xref:Microsoft.ML.Transforms.Text.WordHashBagEstimator> is different from <xref:Microsoft.ML.Transforms.Text.NgramHashingEstimator>
+    /// in a way that the former takes tokenizes text internally while the latter takes tokenized text as input.
```

Comment:
> in a way that the former takes tokenizes text internally while the latter takes tokenized text as input. [](start = 7, length = 105)

ditto: former ... latter...

Reply:
it's already former.. latter...

---
In reply to: [277089489](https://github.com/dotnet/machinelearning/pull/3440#discussion_r277089489) [](ancestors = 277089489)

Reply:
in that the former ...

---
In reply to: [277095332](https://github.com/dotnet/machinelearning/pull/3440#discussion_r277095332) [](ancestors = 277095332,277089489)

Reply:
I don't get your comment

---
In reply to: [277113027](https://github.com/dotnet/machinelearning/pull/3440#discussion_r277113027) [](ancestors = 277113027,277095332,277089489)